### PR TITLE
support utf-8 encode

### PR DESCRIPTION
**Pull request by ChihChengLiang**
don't crash when dealing with utf8 message.

### CODE REVIEW

File: rtmbot.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def output(self):
                     if limiter == True:
                         time.sleep(.1)
                         limiter = False
-                    message = output[1].encode('ascii','ignore')
+                    message = output[1].encode('utf8')
```
Comment (alevkin) [inline rtmbot.py]: Fix:
                    message = output[1]
                    channel.send_message("{}".format(message).decode('utf-8'))

**Comment by logi85** [comment]
would be great if merged.

**Comment by yyar** [comment]
+1

File: rtmbot.py (hunk 1 above)
Comment (yyar) [inline rtmbot.py]: I think message`message = output[1].encode('utf-8', 'ignore')` could be better.

**Comment by jammons** [comment]
Turns out this was somewhat complicated to make it work in both py2.7 and py3.5. I think I have a solution in https://github.com/slackhq/python-rtmbot/pull/46. Feel free to give it a look and add comments if you'd like.

**Pull request closed without merging**